ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -7,7 +7,10 @@ set -o pipefail
 # TODO: move to image
 curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
 
-export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+case "${CLUSTER_TYPE}" in
+aws-china) export AWS_SHARED_CREDENTIALS_FILE=/var/run/aws-china-credential/.awscred;;
+*) export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred";;
+esac
```

Comment:
This should not be necessary, the secret should come from the cluster profile and there is no reason why it should be in a different path in the aws-chine cluster profile

Reply:
@petr-muller per your comments in slack,  I've created a new pr for adding new profiles https://github.com/openshift/release/pull/21734

I have an aws-china credential in `test-credentials/aws-china-credential`, which has been added in vault.ci.openshift.org, how they could be added in cluster profile? 
I saw the `.awscred` in aws profile is https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml#L709-L711 , but what's the `field` and `item`?
Is there a sample code that I can reference? thanks.

Reply:
You can sync your `.awscred` file from your vault into the cluster profile secret (the same you configured in the ci-secret-bootstrap as destination) by setting the secretsync keys. I have submitted a docs PR https://github.com/openshift/ci-docs/pull/184 in a hope to clarify this process:

```
secretsync/target-namespace: "ci"
secretsync/target-name: "cluster-secrets-<name>"
```

Reply:
thanks @petr-muller , I will have try.